FreeTAKServer/controllers/configuration/MainConfig.py

```diff
@@ -32,6 +34,10 @@ class MainConfig:
     except:
         _ip = "0.0.0.0"
 
+    # create the persistence path if it doesn't exist
+    if not os.path.exists(PERSISTENCE_PATH):
+        os.mkdir(PERSISTENCE_PATH)
```

Comment:
We need to be checking the result of `mkdir` and if it fails, then catch the exception. This will allow us to put a friendlier error message in place. This will also help inexperienced users and hopefully lessen the amount of support that we need to do.

Reply:
also addressed with the most recent commit but could you validate the error message in terms of user friendliness